samples/nrf9160/coap_client/README.rst

```diff
@@ -3,12 +3,24 @@
 nRF9160: nRF CoAP Client
 ########################
 
-The nRF CoAP Client sample demonstrates how to receive data from a public CoAP server with an nRF9160 SiP.
+The nRF CoAP Client sample demonstrates the communication between a public CoAP server and an nRF9160 SiP which acts as the CoAP client.
 
 Overview
-*********
+********
 
-The sample connects to a public CoAP test server, sends periodic GET request for a test resource that is available on the server, and prints the data that is received.
+The nRF CoAP Client sample performs the following actions:
+
+#. Connect to the configured public CoAP test server (indicated by the parameter ``CONFIG_COAP_SERVER_HOSTNAME``)
```

Comment:
Would "specified by" or "configured in" be better than "indicated" here?

Reply:
Changed to 'specified by'.